Remotion/Web/Development.WebTesting.ControlObjects/WebButtonControlObject.cs

```diff
@@ -48,7 +48,7 @@ public IStyleInformation StyleInfo
     /// </summary>
     public string GetText ()
     {
-      return Scope.Text.Trim();
+      return Scope[DiagnosticMetadataAttributes.Content].Trim();
```

Comment:
Trim() should not be necessary when we have a dedicated attribute.
```suggestion
      return Scope[DiagnosticMetadataAttributes.Content];
```